src/styles/utils/function.scss

```diff
@@ -0,0 +1,5 @@
+@function calendarWidth ($day-size, $colums-count, $gap) {
+  @return $day-size * $colums-count
+    + $gap * ($colums-count - 1)
+    + 2 * $padding;
+}
```

Comment:
you didn't pass paddings in function parameters, but you use them in calculations